runtime/core/decoder/torch_asr_decoder.h

```diff
@@ -46,6 +47,7 @@ class TorchAsrDecoder {
   // Return true if we reach the end of the feature pipeline
   bool AdvanceDecoding();
   void AttentionRescoring();
+  void PostProcesser();
```

Comment:
change to ProcessBlank()?  and add some comment and example here, such as
// Replace special blank  symbol "▁" (U+2581, \xe2\x96\x81) with white space.
PostProcesser is more than just just blank processing.